lib/util/progress.js

```diff
@@ -20,46 +20,47 @@
  * THE SOFTWARE.
  *
  */
+const Readline = require('readline');
 
-var modulus = require('../modulus');
+const WIDTH = 20;
+const ANIMATE_TIME = 250;
 
 var Progress = {};
 
-Progress.indeterminate = function(fmt, opts) {
+Progress.Indeterminate = function (fmt, opts) {
```

Comment:
these actually don't throw with the updated `mod-standard` so i changed them back.